README.md

```diff
@@ -5,6 +5,28 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
 nvm install 12
 nvm use 12
 npm install --global yarn
-yarn 
+yarn
 yarn develop
-```
\ No newline at end of file
+```
```

Comment:
Perhaps, the author of the assignment hasn't published these changes when it was uploaded to Gdrive and due to that these changes are showing up.